src/pages/product-store/product-store.jsx

```diff
@@ -0,0 +1,18 @@
+import "./product-store.css";
+import Filter from "../../components/filter/filter";
+import { RendingProducts } from "../../components/product-listing/productListing";
+
+
+
+const ProductStore = () => {
+
+
+    return (
+        <div className="productMain-container">
+            <Filter />
+            <RendingProducts />
```

Comment:
remove the unnecessary space,else others code is good

Reply:
you are right but these spaces make the code to look clean